addon/templates/components/columns/base.hbs

```diff
@@ -2,8 +2,8 @@
   {{component column.component column=column table=table tableActions=tableActions sortIcons=sortIcons}}
 {{else}}
   {{label}}
-  {{#if column.sorted}}
-    <i class="lt-sort-icon {{if column.ascending sortIcons.iconAscending sortIcons.iconDescending}}"></i>
+  {{#if column.sortable}}
+    <i class="lt-sort-icon {{get sortIcons sortIcon}}"></i>
```

Comment:
With the other comment regarding `sortIcon` in mind, this would then be:

```hbs
{{#if sortIcon}}
   <i class="lt-sort-icon {{get sortIcons sortIcon}}"></i>
{{/if}}
```